bin/startPipeline.sh

```diff
@@ -450,7 +450,7 @@ fi
 # Parse sample sheets.
 #
 sampleSheets=( "$(ls -1 "${TMP_ROOT_DIR}/Samplesheets/"*".${SAMPLESHEET_EXT}")" )
-for sampleSheet in "${sampleSheets[@]}"
+for sampleSheet in ${sampleSheets[@]}
```

Comment:
Array unquoten is geen fix.

Reply:
nou in dit geval wel

Reply:
je hebt 2 keuzes: je wrapped je ls commando in de regel er boven niet in quotes of je doet het niet bij de for loop.
Als je bij beide quotes gebruikt krijg je dat alles als 1 string gebruikt.
bij SC warning staat ook het volgende:
_Exceptions
When you want globbing of individual elements._

Reply:
De array zou gequote moeten worden bij gebruik. Bij definieren is de `ls` problematisch: alle witruimte - zowel een nieuwe regel als een spatie in een bestandsnaam - zal gebruikt worden om elementen te splitsen. Beter is een readarray i.c.m. met een find zoals bijvoorbeeld hier in `bin/moveSamplesheets.sh`:
```
readarray -t samplesheets < <(find "${samplesheetsSource}" -maxdepth 1 -mindepth 1 -type f -name "*.${SAMPLESHEET_EXT}")
```
Als je samplesheet ook een symlink kan zijn, dan moet je `-type` nog even aanpassen.